### ISSUE TITLE
<span style="color" > could be rendered

### ISSUE THREAD

**Issue by habi** [marktext/marktext#1641]
### Describe your feature request
I'm writing a monthly link list with [this bit of code](https://github.com/habi/pinboard-angelesen).
The template formats the source of the link with `<span style="color: #999999;">($sourceURL)</span>`.
Marktext does not render this in the preview, while other Markdown editors (namely [typora](https://typora.io/)) do (see attached screenshot).
I think it would be nice if the preview would also render this bit of code.

### Additional context [optional]

![Screenshot from 2019-11-07 09-11-26](https://user-images.githubusercontent.com/1651235/68371283-a705d980-013e-11ea-8556-c71038d11c2d.png)

**Comment by Jocs** [comment]
@habi it works fine on my computer
<img width="785" alt="截屏2019-11-07下午6 52 54" src="https://user-images.githubusercontent.com/9712830/68383065-e1e42d80-018f-11ea-961b-5b5d54f044e8.png">

What's your Mark Text version?

**Comment by habi** [comment]
v0.15.0, on Ubuntu 18.04, started as AppImage, as specified [here](https://github.com/marktext/marktext/blob/develop/docs/LINUX.md).

**Comment by Jocs** [comment]
Can you put a `space` before `<span>` and after `</span>` to see whether it is work?

**Comment by habi** [comment]
Doesn't make a difference.
Here's what I copied into a new file (BTW copy-pasting the chunk in the same document does remove the '<span>' tags :) )
````
   asdf <span style="color: #999999;"> asdf </span>
-   [A History of Pizza](https://www.historytoday.com/archive/historians-cookbook/history-pizza) <span style="color: #999999;"> historytoday.com </span> : A History of Pizza | History Today | 
````

With this bit of text I don't get any color in Marktext, in Typora I do, as shown in the screenshot (background Marktext, foreground Typora).

![Screenshot from 2019-11-07 15-17-04](https://user-images.githubusercontent.com/1651235/68396368-c1a57600-0171-11ea-8616-0a28f6657095.png)

**Comment by fxha** [comment]
@habi @Jocs Closing this issue because the feature was not supported in versions before v0.16.0 and is only available on develop/v0.16.0 soon. Please note that styled text is not official supported by markdown and may fail with other parsers like GitHub.
